pages/common/spatial.md

```diff
@@ -0,0 +1,35 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Run this when you use a project for the first time:
+
+`spatial worker build`
+
+- Build workers for local deployment on Unity on on Osx:
```

Comment:
"on" is repeated. And also, please change Osx to macOS.

Reply:
I think my latest check in fixes this [link](https://github.com/tldr-pages/tldr/pull/1652/commits/e953129676feb85c66907c12f31ecc3447328cb6)